config/src/main/scala/bloop/config/Config.scala

```diff
@@ -234,6 +234,7 @@ object Config {
       workspaceDir: Option[Path],
       sources: List[Path],
       sourcesGlobs: Option[List[SourcesGlobs]],
+      sourceRoots: Option[List[String]],
```

Comment:
Should this be `Option[List[Path]]`?

Reply:
Even if bloop doesn't need this typed as a `Path`, I believe it's more readable for clients using bloop-config that this should be a path  and not a random string. Additionally, `Path` uses structural sharing resulting in lower memory usage.